src/Layout/ErrorReporter/index.tsx

```diff
@@ -26,8 +26,7 @@ export class ErrorReporter extends React.PureComponent<Props> {
       <Page>
         <PageSection
           isFilled={true}
-          padding={{ default: 'noPadding' }}
-          className={styles.backdropBackground}
+          className={styles.backdropBackground + ' pf-c-page-section-no-padding'}
```

Comment:
Property `padding={{ default: 'noPadding' }}` works seamlessly for me. Did you have any issues with that?

Reply:
No issues with it, it just slightly changes the way the UI looks. If that's fine then I can revert the ErrorReporter change

Reply:
Hmm, I don't see any changes in UI between using property and CSS class. So, I'd like to revert changes in this component.